src/commonMain/kotlin/exchange.dydx.abacus/calculator/MarginCalculator.kt

```diff
@@ -202,15 +203,13 @@ internal object MarginCalculator {
         subaccount: Map<String, Any>?,
         tradeInput: Map<String, Any>?,
     ): Boolean {
-        val marketId = parser.asString(tradeInput?.get("marketId")) ?: return true
-        val position = parser.asNativeMap(parser.value(subaccount, "openPositions.$marketId"))
-        val currentSize = parser.asDouble(parser.value(position, "size.current")) ?: 0.0
-        val postOrderSize = parser.asDouble(parser.value(position, "size.postOrder")) ?: 0.0
-        return postOrderSize.abs() > currentSize.abs()
+        return getPositionSizeDifference(parser, subaccount, tradeInput)?.let {
+            it.compareTo(Numeric.double.ZERO) == 1
```

Comment:
From the kotlin [docs](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-comparable/compare-to.html), i can't tell if the comparison to 1 will work. would it be safer to `> 0` in this case?

Reply:
if we know it will be 1 or -1 then I defer to you 😄